benchmark/kvbench/test/custom_traffic_perftest.py

```
@@ -123,44 +299,120 @@ def __init__(
+            )
+            return
+        # Skip remote metadata when running single-rank or when no remote-capable backend is available
+        if self.world_size == 1:
+            logger.debug(
+                "[Rank %d] Single-rank run, skipping metadata exchange", self.my_rank
+            )
+            return
+        logger.debug("[Rank %d] Getting local agent metadata...", self.my_rank)
+        try:
+            md = self.nixl_agent.get_agent_metadata()
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)